modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionsExchangeFuture.java

```diff
@@ -723,6 +725,25 @@ private void initCachesOnLocalJoin() throws IgniteCheckedException {
         }
 
         cctx.cache().startCachesOnLocalJoin(caches, initialVersion());
+
+        ensureClientCachesStarted();
+    }
+
+    /**
+     * Start client caches if absent.
+     */
+    private void ensureClientCachesStarted() {
+        GridCacheProcessor cacheProcessor = cctx.cache();
+        List<CacheConfiguration> notStartedCacheConfigs = new ArrayList<>();
```

Comment:
Please introduce blank line before and after this statement as per Ignite code conventions.